jekyll/_cci2/prevent-unregistered-users-from-spending-credits.adoc

```diff
@@ -0,0 +1,43 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Prevent unregistered users from spending credits
+:page-layout: classic-docs
+:page-liquid:
+:page-description: A how to guide to configuring your org so that unregistered users will not be able to perform actions that spend credits3
```

Comment:
Is the `3` on `credits3` a typo or part of the markup?

Reply:
typo! thanks